code/ui/manager/src/components/sidebar/Search.tsx

```diff
@@ -67,15 +78,17 @@ const SearchIconWrapper = styled.div(({ theme }) => ({
 const SearchField = styled.div({
   display: 'flex',
   flexDirection: 'column',
+  flexGrow: 1,
   position: 'relative',
 });
 
 const Input = styled.input(({ theme }) => ({
   appearance: 'none',
-  height: 32,
+  height: 28,
   paddingLeft: 28,
   paddingRight: 28,
-  border: `1px solid ${theme.appBorderColor}`,
+  border: 0,
+  boxShadow: `${theme.button.border} 0 0 0 1px inset`,
```

Comment:
Paired on this with @cdedreuille

Reply:
Why was the height reduced to 28px? In the VTA we have this, which is 32px and very similar:
<img width="188" alt="Screenshot 2024-04-12 at 14 00 23" src="https://github.com/storybookjs/storybook/assets/321738/d6c56fb7-b131-4a9a-89e3-cdaa49426a26">

If we're going to stick with 28px (not my preference), the vertical alignment of the `/` needs to be fixed:
<img width="214" alt="Screenshot 2024-04-12 at 13 59 34" src="https://github.com/storybookjs/storybook/assets/321738/44bf286e-4bee-4184-bbd1-190be17c0eae">

I prefer 28px because it doesn't look as cramped.

Reply:
@ghengeveld The button you are referring to is the only one with that size and it appears only on portrait mode. If you switch to landscape, it goes back to 28px. Generally speaking we are using 28px quite often in the app. The reason why we chose to go with 28px here is to be consistent with the other icons around.

That said, I do agree that we should align its content vertically.

Changing the search background is a nice idea but I would keep it separate please. We have to consider it with the rst of the theming updates we want to do.

Reply:
Yeah feel free to ignore the background comment, I recognize this isn't the place for it.